explore/src/main/scala/explore/common/GroupQueries.scala

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.common
+
+import cats.effect.Async
+import cats.syntax.all.*
+import clue.FetchClient
+import clue.data.syntax.*
+import eu.timepit.refined.types.numeric.NonNegShort
+import explore.DefaultErrorPolicy
+import lucuma.core.model.Group
+import lucuma.schemas.ObservationDB
+import lucuma.schemas.ObservationDB.Types.*
+import queries.common.GroupQueriesGQL.*
+
+object GroupQueries:
+
+  /**
+   * @param groupId
+   *   Group to move
+   * @param parentGroup
+   *   New parent group, `None` to move to top level
+   * @param parentGroupIndex
+   *   New index in parent group, `None` to leave position unchanged
```

Comment:
What does it mean to leave the index `unchanged` if it is a new parent group?

Reply:
'Unchanged' is definitely the wrong word. It means "let the odb figure out the index". Though in reality this is always filled (primereact-tree gives always gives the index)